docs/web-chat/api.mdx

```diff
@@ -137,6 +137,151 @@ You should call this API before a conversation starts. In other words, it works
 
 </Admonition>
 
+## Login {#login}
+
```

Comment:
In this new section, we need to mention that it is in Beta and to enable touch base with Customer Success. Mention it as a note separately.